.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,28 @@
+# Workflow name
+name: 'Chromatic Publish'
+
+# Event for the workflow
+on: push
```

Comment:
on push of which branch? Kindly mention it out.

Also, do we need chromatic URL for PR deploys as well?

Reply:
Done.

I am sorry, I didn't understand `do we need chromatic URL for PR deploys as well?`. Can you please elaborate?